src/pybind/mgr/dashboard/frontend/src/defaults.scss

```diff
@@ -1,11 +1,46 @@
-$warning-background-color: #fff3cd;
+/*The file suppose to contain all variables, mixins and extends*/
+
+/*Health status*/
+$color-solid-red: #ff0000;
+$color-orange: #ff6600;
+$color-bright-yellow: #ffc200;
+$color-bright-green: #00bb00;
+
+/*Login Form*/
+$color-light-gray: #ececec;
+$color-mild-gray: #555555;
+$color-dark-gray: #474544;
 $oa-color-blue: #288cea;
-$oa-color-light-blue: #afd9ee;
+$color-cornflower-blue: #66afe9;
+
+/*Table Component*/
+$color-off-white: #f5f5f5;
+$color-mild-white: #f6f6f6;
+$border-color-white: 1px solid rgba(0,0,0,.09);
```

Comment:
I think you should split that and set the border style in the table component scss.

Reply:
This is nice step to put all color literals into a single place, but pushing things a bit further: what about using a color naming scheme that describe the meaning/intention of the element colored instead of simply describing the color shade?

Let me put an example: instead of `color-solid-red`, what about `color-health-error`?

Reply:
Sounds good to me. But i would like to see something like this to make it really really simple:

```
$color-solid-red: #FF0000;
$color-health-error: $color-solid-red;
```

Reply:
Ack